File: src/com/anathema_roguelike/entities/characters/foes/corruptions/Corruption.java
package com.anathema_roguelike.entities.characters.foes.corruptions;

import com.anathema_roguelike.entities.characters.foes.traits.Trait;
import com.anathema_roguelike.entities.characters.perks.PerkGroup;

public abstract class Corruption extends PerkGroup {
	
	public Corruption(Trait<?> ...traits) {
		super(traits);
	}
}

File: src/com/anathema_roguelike/entities/characters/foes/corruptions/Corruption.scala
package com.anathema_roguelike
package entities.characters.foes.corruptions

import com.anathema_roguelike.entities.characters.foes.traits.Trait
import com.anathema_roguelike.entities.characters.perks.PerkGroup

abstract class Corruption(val traits: Trait[_]*) extends PerkGroup(traits:_*) {
}